src/Microsoft.DotNet.XHarness.WebAssembly/xunit-runner.cs

```diff
@@ -0,0 +1,367 @@
+                            }
+
+                            ParameterInfo[] pars = method.GetParameters();
+                            for (int i = 0; i < dataRow.Length; ++i)
+                                dataRow[i] = ConvertArg(dataRow[i], pars[i].ParameterType);
+                            method.Invoke(obj, BindingFlags.Default, null, dataRow, null);
+                        }
+                    }
+                    catch (Exception ex)
+                    {
+                        Console.WriteLine("FAIL: " + tc.DisplayName + " " + ex);
+                        failed = true;
+                    }
+                }
+                if (failed)
+                {
+                    nfail++;
+                    return;
+                }
+            }
+            else
+            {
+                if (!string.IsNullOrEmpty(tc.SkipReason))
+                {
+                    nskipped++;
+                    return;
+                }
+                nrun++;
+                string name = tc.DisplayName;
+                int indx = name.IndexOf("(");
+                if (indx != -1)
+                    name = name.Substring(0, indx);
+                if (name != _last_name)
+                    Console.WriteLine(name);
+                _last_name = name;
+                try
+                {
+                    object obj = null;
+                    if (!method.IsStatic)
+                    {
+                        ConstructorInfo constructor = method.ReflectedType.GetConstructor(Type.EmptyTypes);
+                        if (constructor != null)
+                            obj = constructor.Invoke(null);
+                        else
+                            obj = System.Runtime.Serialization.FormatterServices.GetUninitializedObject(method.ReflectedType);
+                    }
+                    object[] args = tc.TestMethodArguments;
+                    if (args != null)
+                    {
+                        ParameterInfo[] pars = method.GetParameters();
+                        for (int i = 0; i < args.Length; ++i)
+                        {
+                            args[i] = ConvertArg(args[i], pars[i].ParameterType);
+                        }
+                    }
+                    method.Invoke(obj, args);
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine("FAIL: " + tc.DisplayName);
+                    Console.WriteLine(ex);
+                    nfail++;
+                }
+            }
+        }
+    }
+
+    class CmdLineParser : CommandLine
+    {
+        public CmdLineParser(string[] args) : base(args, s => true)
+        {
+        }
+    }
+
+    public class XunitDriver
+    {
+        static WasmRunner s_testRunner;
+
+        static int Main(string[] args)
+        {
+            // Process rsp files
```

Comment:
We've stopped relying on rsp files and instead are using attributes.  I'd get rid of the comment since it's not a big priority.

Reply:
done